### ISSUE TITLE
kismatic crashs

### ISSUE THREAD

**Issue by marclijour** [apprenda/hands-on-with-kubernetes-workshop#2]
There seems to be a problem with the executable?

marc@marc-VPCZ2290S ~/Devel/hands-on-with-kubernetes-workshop $ make provision-cluster
chmod 600 ssh/cluster.pem
time ./kismatic install apply -f kismatic-cluster.yaml
./kismatic: 1: ./kismatic: ����: not found
./kismatic: 2: ./kismatic: Syntax error: ")" unexpected
Command exited with non-zero status 2
0.00user 0.00system 0:00.00elapsed 0%CPU (0avgtext+0avgdata 1628maxresident)k
0inputs+0outputs (0major+75minor)pagefaults 0swaps
Makefile:17: recipe for target 'provision-cluster' failed
make: *** [provision-cluster] Error 2

![screenshot from 2017-02-06 21-48-21](https://cloud.githubusercontent.com/assets/4466879/22675895/38e87380-ecb6-11e6-945b-5b932726b75b.png)

**Comment by swade1987** [comment]
@marclijour if my memory serves me correctly you were running from a linux machine, correct?

You may have downloaded the darwin binary instead of the Linux one.

If you execute the following ...

`rm -rf ansible cfssl generated runs kismatic kubeconfig provision`

Then execute ...

`wget -O - https://github.com/apprenda/kismatic/releases/download/v1.2.1/kismatic-v1.2.1-linux-amd64.tar.gz | tar -zx`

Now execute `make provision-cluster` that should hopefully fix it.

**Comment by marclijour** [comment]
Can you remind me how to generate the droplets on Digital Ocean? I trashed them after the meetup. The doc has changed so much I can't find the where to pick up the process.

**Comment by swade1987** [comment]
@marclijour sorry for the delay in responding. You're using a MacBook correct?

**Comment by marclijour** [comment]
Linux

**Comment by swade1987** [comment]
Perfect, follow the steps for Mac and you should be fine.
Basically we create a bootstrap node and execute the commands from that machine.
Let me know how you get on.

**Comment by marclijour** [comment]
OK. I made it work on my Linux machine. I proposed a couple of fixes (pull requests) to use the correct directory on the bootstrap node. 

There is another comment I want to make here. After the "make provision-cluster" is done, it suggests to run "kismatic dashboard" to view the Kubernetes dashboard. However, it won't open a browser through ssh like that. A work around would be to install a browser like firefox on the bootstrap node, and to run it through the dev machine with a command line this: ssh -i ssh/cluster.pem -X root@<bootstrap-node-IP> firefox <link-generated> &
However, it might be faster to execute the browser locally, on the dev laptop.

On the Kubernetes Dashboard, I got to verify that Kubernetes service is OK. Now, there is no apps deployed. It would be nice to have some demo apps to run.
